karrot/groups/tasks.py

```diff
@@ -1,16 +1,15 @@
-from anymail.exceptions import AnymailAPIError
 from datetime import timedelta
```

Comment:
How come you removed this line? Seems unrelated?

```py
stats_utils.periodic_task('group__mark_inactive_groups', seconds=t.elapsed_seconds)
```

(GitHub wouldn't let me comment on the actual removed line...)

Reply:
Not sure why this shows up, I didn't change anything about this line. It's actually the last line in the file, but it shows up as line number zero? Might be a problem with github...